[Feature] Add optional route tenant prefix (#7655)
laravel-filament/filament: packages/panels/docs/10-tenancy.md

}
```

## Configuring the tenant route prefix

> By default the URL structure will put the tenant id or slug immediately after the panel path. If you wish to prefix the tenant id or slug with a value to tenant routes you may do so by using the `tenantRoutePrefix()` method.

Reviewer: ```suggestion
By default the URL structure will put the tenant ID or slug immediately after the panel path. If you wish to prefix it with another URL segment, use the `tenantRoutePrefix()` method:
```

```php
use Filament\Panel;

public function panel(Panel $panel): Panel
{
    return $panel
        // ...
        ->path('admin'}
        ->tenant(Team::class)
        ->tenantRoutePrefix('team');
}
```

> Without this the URL structure will be `https://example.com/admin/1` for tenant id 1, but with the route prefix it will become `https://example.com/admin/team/1`.

Reviewer: ```suggestion
Before, the URL structure was `/admin/1` for tenant 1. Now, it is `/admin/team/1`.
```
